### DISCUSSION TITLE
trafficsplit based on sourcelabels? like istio

### DISCUSSION THREAD

**Discussion by hawk-liu** [linkerd/linkerd2#7081] [Q&A] [answered]
we are trying linkerd for our app.
but we want serviceA-canary->serviceB-canary->serviceC-canary, not serviceA-canary->serviceB-pro->serviceC-pro. 
the doc only show trafficsplit based http header/weight, is there any way like istio sourcelabel solution?

**Comment by adleong** [comment, marked as answer]
Hi @hawk-liu!  Linkerd does not support trafficsplitting based on source labels.  If this is a feature that you're interested in, I'd recommend filing a detailed issue.  Make sure to describe the problem that that you're using trafficsplitting to solve with as much context as possible (so that we can also explore alternate solutions).

**Comment by hawk-liu** [comment]
ok, thanks, i'll create a issue about this
